fxgl/src/main/kotlin/com/almasb/fxgl/app/LoopRunner.kt

```diff
@@ -94,29 +93,35 @@ internal class LoopRunner(
     }
 
     private fun frame(now: Long) {
-        if (lastFPSUpdateNanos == 0L) {
+        if (lastFrameNanos == 0L) {
+            lastFrameNanos = now
             lastFPSUpdateNanos = now
-            fpsBuffer2sec = 0
+            fpsSamplingCount = 0
         }
 
-        cpuNanoTime = measureNanoTime {
-            runnable(tpf)
+        tpf = (now - lastFrameNanos).toDouble() / 1_000_000_000
+
+        val ticksPerSecond = if (ticksPerSecond < 0) 60 else ticksPerSecond // For JavaFX loops, cap at 60fps too
```

Comment:
Please note that this is a safety mesure. I was supprised to see that JavaFX was defaulted to 120FPS on a Mac (instead of the 60FPS defined in the JavaFX documentation). In order to make tests "predictable", I had to cap it to 60FPS as each LoopRunner { t += it; Thread.sleep(lag) } test cases were failing by returning 120FPS